api/api-rules/violation_exceptions.list

```diff
@@ -643,6 +643,7 @@ API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,K
 API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,KubeControllerManagerConfiguration,HPAController
 API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,KubeControllerManagerConfiguration,JobController
 API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,KubeControllerManagerConfiguration,KubeCloudShared
+API rule violation: names_match,k8s.io/kube-controller-manager/config/v1alpha1,KubeControllerManagerConfiguration,MizarArktosNetworkController
```

Comment:
what was the reason this exception of rule check needed? just curious.

Reply:
co-ask.

Reply:
I believe it is because of type KubeControllerManagerConfiguration was added MizarArktosNetworkController in pkg/controller/apis/config/types.go